### PR TITLE
Improve the Typescript template.

### PR DESCRIPTION
**Pull request by meotimdihia**
The problem was said in the issue: https://github.com/fastify/fastify-cli/issues/591

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
CI is failing

File: templates/app-ts/src/app.ts [hunk 1]
```diff
@@ -1,11 +1,8 @@
 import { join } from 'path';
 import AutoLoad, {AutoloadPluginOptions} from '@fastify/autoload';
-import { FastifyPluginAsync } from 'fastify';
-
-export type AppOptions = {
-  // Place your custom options for app below here.
-} & Partial<AutoloadPluginOptions>;
+import { FastifyPluginAsync, FastifyServerOptions } from 'fastify';
 
+export type AppOptions = FastifyServerOptions & Partial<AutoloadPluginOptions>;
```
Comment (climba03003) [inline templates/app-ts/src/app.ts:5]: It is not extendable as the previous one. Using `interface ... extends` would be better.

```ts
interface AppOptions extends FastifyServerOptions, Partial<AutoloadPluginOptions> {

}
```

Comment (meotimdihia) [reply templates/app-ts/src/app.ts]: @climba03003 thanks, I have updated the code.

**Review by climba03003 — APPROVED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**